TRAJECTORY_SEQUENCE = {
    1: "redBall1_pickPose",
    2: "redBall_placePose",
    3: "redBall2_pickPose",
    4: "redBall_placePose",
    5: "blueBall1_pickPose",
    6: "blueBall_placePose",
    7: "blueBall2_pickPose",
    8: "blueBall_placePose",
}

PICK_OFFSET = {
    'x': 0,
    'y': 0,
    'z': 0.065,
}

PLACE_OFFSET = {
    'x': 0.1,
    'y': 0,
    'z': 0.1,
}